supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -11,5 +11,31 @@
 [[admonitions]]
 == Admonitions
 
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions in close proximity to one another. If multiple admonitions are necessary, restructure the information by removing the less important statements and placing them into the flow of the main content.
+
+Allowed admonition types:
+
+- NOTE
+- IMPORTANT
+- WARNING
+- TIP
+
+[IMPORTANT]
+====
+CAUTION, which is another type of AsciiDoc admonition, is not fully supported by the Red Hat Customer Portal. This admonition should not be used.
```

Comment:
Passive voice: This admonition should not be used.

Suggestion `Do not use this admonition type.`

Reply:
+1